component/src/main/java/io/siddhi/extension/execution/unitconversion/length/MetreToMillimetre.java

```diff
@@ -47,17 +48,18 @@
         parameters = @Parameter(
                 name = "p1",
                 description = "The value that needs to be converted from meters into millimeters.",
-                type = {DataType.INT, DataType.LONG, DataType.FLOAT, DataType.DOUBLE}),
+                type = {DataType.INT, DataType.LONG, DataType.FLOAT, DataType.DOUBLE},
+                dynamic = true
+        ),
+        parameterOverloads = {
+                @ParameterOverload(parameterNames = {"p1"})
+        },
         returnAttributes = @ReturnAttribute(
                 description = "The value returned in millimeters.",
                 type = {DataType.DOUBLE}),
         examples = @Example(
-                syntax = "define stream UnitConversionForMetreToMillimetreStream (inValue int); \n" +
-                        "from UnitConversionForMetreToMillimetreStream \n" +
-                        "select unitconversion:mTomm(inValue) as UnitConversionValue \n" +
-                        "insert into OutMediationStream;",
-                description = "The value in meters from 'UnitConversionForMetreToMillimetreStream' input stream" +
-                        "is converted into millimeters and returned to the 'OutMediationStream' output stream."
+                syntax = "unitconversion:mTomm(inValue) as UnitConversionValue \n",
```

Comment:
Better to remove `as UnitConversionValue` also it's better to have actual value instead of `inValue` and say the output will be `x`.

Reply:
Updated in https://github.com/siddhi-io/siddhi-execution-unitconversion/pull/32/commits/75b16141f60ee00b208ee8050212e2967ae912e4 ...